### ISSUE TITLE
Low space on rancher setup causes containers to malfunction

### ISSUE THREAD

**Issue by rtud** [cami-project/cami-project#203]
## Why
Following a quick investigation before starting work on the Google Calendar integration, we have found that the API is no longer functioning properly.

## What
* [x] the cause for the API malfunction is investigated and follow-up issues are created to resolve it
* [x] on successful debug, a separate issue is created for preventing the problem from manifesting again

## Notes
The investigation I have done alongside @cretualexandru, indicated that the cause might be low space left on the rancher machine caused by either:
   * unused docker images previously generated and left there
   * logging messages that piled up

**Comment by cretualexandru** [comment]
It seems that this is a classic docker issue. I managed to quickly fix it running two commands:

Delete all dangling images
```
docker rmi $(docker images -qf dangling=true)
```

Delete all dangling volumes
```
docker volume rm $(docker volume ls -qf dangling=true)
```

It seems that Rancher does not like when images are manually deleted, so I also rebooted both master and slave machines in order to get Rancher running again, but I think this could have been done by only restarting rancher container.

Other useful info that we can use to setup an auto-cleanup process:
https://www.calazan.com/docker-cleanup-commands/
https://github.com/chadoe/docker-cleanup-volumes

**Comment by rtud** [comment]
I have documented the quick fix procedure inside the main [README.md](https://github.com/cami-project/cami-project/blob/master/README.md), and filed follow-up issue #209 for looking for a proper solution to this problem. Closing, thank you!
